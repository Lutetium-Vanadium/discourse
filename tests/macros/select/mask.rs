fn main() {
    let q = discourse::questions![Select { mask: todo!() }];
}
