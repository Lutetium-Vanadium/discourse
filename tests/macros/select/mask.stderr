error: option `mask` does not exist for kind `select`
 --> $DIR/mask.rs:2:44
  |
2 |     let q = discourse::questions![Select { mask: todo!() }];
  |                                            ^^^^
